bin/varnishd/cache/cache_vcl.c

```diff
@@ -85,6 +86,8 @@ VCL_Bo2Ctx(struct vrt_ctx *ctx, struct busyobj *bo)
 	ctx->sp = bo->sp;
 	ctx->now = bo->t_prev;
 	ctx->ws = bo->ws;
+	ctx->handling = &bo->wrk->handling;
+	*ctx->handling = 0;
```

Comment:
Isn't this already done by vcl_call_method() in cache_vrt_vcl ?

Reply:
`vcl_call_method()` uses a local `struct vrt_ctx`, as does `VDI_GetHdr()`, so there is no effect of the former on the latter.
But you rightly pointed out that my suggestion duplicates these two lines, I will check if they could be removed from `vcl_call_method()`.

Reply:
sorry, I was looking at master, I actually had already suggested to remove the duplication here:
https://github.com/varnishcache/varnish-cache/pull/2998/commits/218c818023dfb3446e413dd00b64c755be764876#diff-f00ee27228a92dc1471fc250affb40c9L444